Documentation.app/Contents/MacOS/Library.playground/Pages/APIs.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,57 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: Modules
+ */
+// nef:end
+
+/*:
+ ## APIs
+
+ `nef` library provides you the whole features into your macOS project. It enables you to build new tooling on top of nef.
+ 
+ It gives you an easy access to render Markdown files, Jekyll content and Carbon images in a functional way. Also you can compile or clean your nef Playrounds, even make an Xcode Playground/Playground Book compatible with 3rd-party libraries.
+
+ ### Playground
+
+ Xcode Playgrounds are a friendly tool for prototyping and trying new concepts. However, they have some limitations, if you need to create playgrounds with 3rd-party libraries support or make your Xcode Playground compatibles with the rest of the options.
+ 
+ You can find more information in [Playground API docs](TODO).
+ 
+ ### Compile
+
+ If you need to compile a nef Playground in the same way you will do from Xcode Playground IDE.
+ 
+ You can find more information in [Compile API docs](TODO)
+
+ ### Clean
+
+ In case you need to reset a nef Playground, for example to share as light as possible (without any extra compilation file).
+ 
+ You can find more information in [Clean API docs](TODO)
+
+ ### Markdown
+
+If you need to render the content of a Playground page, or anything else that combine [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
```

Comment:
What would be the *anything else* that is not a Playground page?

```suggestion
This API lets you render the content of a Playground page, or anything else that contains [**markup formatting**](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) and code.
```